src/main/java/cz/muni/ics/oidc/ToOidcSynchronizer.java

```diff
@@ -358,7 +358,7 @@ private void setRefreshTokens(MitreidClient c, Map<String, PerunAttributeValue>
                 c.getScope().add(OFFLINE_ACCESS);
                 c.getGrantTypes().add(GRANT_REFRESH_TOKEN);
                 c.setClearAccessTokensOnRefresh(true);
-                c.setReuseRefreshToken(true);
+                c.setReuseRefreshToken(false);
```

Comment:
This might present a breaking change for somebody (who relies on reusing the same refresh token without updating it with the new one). I agree with the change, but service owners should be notified.

Reply:
Good point, but I would not care too much about that. I would say it is their "bad" implementation... :)

Reply:
It cannot be "bad" if they are following the standard - releasing new refresh token is optional per [RFC 6749](https://datatracker.ietf.org/doc/html/rfc6749#section-1.5)

Reply:
Updated as a breaking change then :). 👍🏼